events/schema.py

```diff
@@ -275,6 +279,42 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return UnenrolOccurrenceMutation(child=child, occurrence=occurrence)
 
 
+class SetOccurrenceAttendance(graphene.relay.ClientIDMutation):
+    class Input:
+        occurrence_id = graphene.GlobalID()
+        child_id = graphene.GlobalID()
+        attended = graphene.Boolean(
+            description="This field is required (but it can be null)."
+        )
+
+    occurrence = graphene.Field(OccurrenceNode)
+    child = graphene.Field(ChildNode)
+
+    @classmethod
+    @login_required
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **kwargs):
+        occurrence = get_obj_if_user_can_administer(
+            info, kwargs["occurrence_id"], Occurrence
+        )
+        child = get_obj_if_user_can_administer(info, kwargs["child_id"], Child)
+
+        if "attended" not in kwargs:
+            raise DataValidationError('"attendance" is required.')
```

Comment:
Maybe `"attended" is required.` should make more sense here

Reply:
Fixed.